src/widget/wknob.cpp

```diff
@@ -45,30 +46,6 @@ void WKnob::wheelEvent(QWheelEvent* e) {
     m_handler.wheelEvent(this, e);
 }
 
-void WKnob::guiTick() {
-    mixxx::Duration now = mixxx::Time::elapsed();
-    if (now - m_lastActivity > mixxx::Duration::fromSeconds(1)) {
-        m_guiTickTimer.stop();
-    }
-    if (m_lastActivity > m_lastRender) {
-        update();
-        m_lastRender = m_lastActivity;
-    }
-}
-
 void WKnob::inputActivity() {
-    // Bug #1793015: With Qt 4, we would simply call QWidget::update in response
-    // to input events that required re-rendering widgets, relying on Qt to
-    // batch them together and deliver them at a reasonable frequency. On macOS,
-    // the behavior of QWidget::update in Qt 5 seems to have changed such that
-    // render events happen much more frequently than they used to. To address
-    // this, we instead use a downsampling timer attached to the VSyncThread's
-    // render ticks for the waveform renderers. The timer invokes guiTick(),
-    // which is responsible for actually calling QWidget::update(). When input
-    // arrives, we call inputActivity to attach the timer. After 1 second of
-    // inactivity, we disconnect the timer.
-    m_lastActivity = mixxx::Time::elapsed();
-    if (!m_guiTickTimer.isActive()) {
-        m_guiTickTimer.start(mixxx::Duration::fromMillis(20));
-    }
+    m_renderTimer.activity();
```

Comment:
This function is no longer necessary. Just call `m_renderTimer.activity()` where `inputActivity()` is now.

Reply:
Yea, I considered it, but I'd rather not touch private variables. I never liked how KnobEventHandler/SliderEventHandler are friends of WKnob/WKnobComposerd/WSliderComposed.

Reply:
Oh yeah, I forgot these functions are called from Knob/SliderEventHandler, not directly from the widget classes.